### ISSUE TITLE
[2.1.4] iPhone 6 Plus / 10.1.1 - Error message "Invalid URL" is displayed while playing motif.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#1228]
Frequency: Every time

Action: 
1: Launch the app and login with valid credentials.
2: Tap on "START" and navigate to "Motifs" tab.
3: Expand the "Trippy" category and play the highlighted motif in the screenshot.
4: Check if motif preview is displayed.

Expected result: Motif preview should be displayed after tapping on "Play" option.

Actual result: Error message "Invalid URL" is displayed while playing motif.

Refer attached screenshot for more details:
![invalid_url](https://cloud.githubusercontent.com/assets/18608449/21451588/30d2dda6-c926-11e6-9e2d-5553fde778b5.jpg)

![invalid_url_1](https://cloud.githubusercontent.com/assets/18608449/21451599/3ab327fe-c926-11e6-977a-28638e1ef4b9.jpg)

**Comment by Pankaj-Testershub** [comment]
@xiuie I can reproduce this issue for build [2.1.4] on iPhone 6S / 10.2

**Comment by hiteshbheda** [comment]
@xiuie Duplicate of #1050 

I can reproduce this issue for build [2.1.4] on iPhone 6S / iOS 10.1.1

**Comment by xiuie** [comment]
@testershub @Pankaj-Testershub @hiteshbheda Have you noticed this problem occuring with other Motifs? Want to know how prevalent it is which will bump up its importance as a bug..

**Comment by hiteshbheda** [comment]
@xiuie No, probably this problem occurs with this motif only. If I'll encounter same problem for other motifs, then will inform you.

**Comment by testershub** [comment]
@xiuie No, issue is specifically observed for highlighted motif only.

**Comment by testershub** [comment]
@xiuie Similar issue is observed for motifs in "Sad","beach","Summer","birthday","dance","City","Food" and "New York"  categories.

Please see issue is observed for motifs which has play icon on it.

**Comment by testershub** [comment]
@xiuie I can still reproduce this issue for build [2.1.12] on iPhone 7 / 10.2.
Issue is reproducible for "weekend" motif category.
